src/helpers/Templates.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Generate html templates for email notifications
+ * @class Templates
+ */
+class Templates {
+  /**
+   * A method for creating a template for notifying users of request for resetting password
+   * @param { string } link - The link to be sent to user email
+   * @returns { string } HTML template
+   * @memberof Templates
+   */
+  static forgotPassword(link) {
+    return `<p>You have requested to reset your password. 
+    click on the link below to change your password.
+    Kindly ignore if this request was not made by you.</p>
+    <a href=${link}>Reset my password</a> <br>
+    <a href=${link}>${link}</a> <br>
+    <p>Best Regards,</p>
+    <p>Authors Haven</p>
+    `;
+  }
+
+  /**
+   * A method for creating a template for notifying users after resetting password
+   * @returns { string } HTML template
+   * @memberof Templates
+   */
+  static resetPassword() {
+    return `<p>Your Password was changed successfully</p> <br> <br>
+    <p>Best Regards,</p>
+    <p>Authors Haven</p>
+    `;
+  }
+}
+
```

Comment:
This is a template email that is going to a user, it needs to be styled.

Reply:
Resolved- I have styled the templates.